client/src/components/Lists/IssueInterests/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC, ReactElement } from 'react';
+import { IIssueAggregate, IPitch } from 'ssw-common';
+
+import { PitchInterests } from '..';
+
+interface IssueInterestsProps {
+  issue: IIssueAggregate;
+}
+
+const IssueInterests: FC<IssueInterestsProps> = ({ issue }): ReactElement => (
+  <>
+    {issue.aggregated.pitches.map((pitch, index) => (
+      <PitchInterests pitch={pitch as Pick<IPitch, 'topics'>} key={index} />
```

Comment:
This is kind of silly because you are still passing the entire pitch object into the Pitch Interests. We could probably consolidate the UserInterests and PitchInterests ones because I am assuming they do the same exact thing. And then instead of doing this `Pick...` thing we just pass in an array of string ids.